resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/utils/MetricNames.java

```diff
@@ -5,6 +5,8 @@
     public static final String SUCCESSFUL = "successful";
     public static final String FAILED = "failed";
     public static final String SLOW = "slow";
+    public static final String SLOW_SUCCESS = "slow_successful";
```

Comment:
I think after renaming the core metric name to `.slow.calls` we could reuse the existing tags like `failed` and `slow`.